src/xdg-desktop-portal-gtk.c

```diff
@@ -85,20 +85,23 @@ on_bus_acquired (GDBusConnection *connection,
                  gpointer         user_data)
 {
   GError *error = NULL;
+  GSettings *lockdown;
 
-  if (!file_chooser_init (connection, &error))
+  lockdown = g_settings_new ("org.gnome.desktop.lockdown");
```

Comment:
This does end up leaked.

Reply:
Yeah, could make it static to make this more obvious.
Or I could make the individual portals take refs (and leak those instead)